pact/Pact/Core/SizeOf.hs

```diff
@@ -436,13 +438,44 @@ instance (SizeOf t, SizeOf i) => SizeOf (DefSchema t i)
 instance (SizeOf n, SizeOf i) => SizeOf (DefTable n i)
 instance (SizeOf n, SizeOf t, SizeOf b, SizeOf i) => SizeOf (Def n t b i)
 
-instance (SizeOf n, SizeOf t, SizeOf b, SizeOf i) => SizeOf (Module n t b i)
+instance (SizeOf n, SizeOf t, SizeOf b, SizeOf i) => SizeOf (Module n t b i) where
+  sizeOf i ver m = fmap (constructorCost 1 +) $ case m of
+    Module
+      { _mName
+      , _mGovernance
+      , _mDefs
+      , _mBlessed
+      , _mImports
+      , _mImplements
+      , _mHash
+      , _mTxHash
+      , _mInfo
+      } -> sum <$> sequence
+        [ sizeOf i ver _mName
```

Comment:
I'm not particularly sure how to feel about these changes tbh. The idea of the gas charge of infos is meant to actually charge on the memory occupied by the module. Moreover, it doesn't exactly change that `sizeOf i ver _mDefs` is still charging for infos.